Dockerfile

```
@@ -1,7 +1,7 @@
 # syntax=docker/dockerfile:1
 
-ARG CUDA_VER=12.0.1
-ARG PYTHON_VER=3.11
+ARG CUDA_VER=unset
+ARG PYTHON_VER=unset
 ARG LINUX_DISTRO=ubuntu
 ARG LINUX_DISTRO_VER=22.04
```

This need not be done in this PR. Just the discussion above made me wonder

Should we do the same thing for the Linux version?

```suggestion
ARG LINUX_DISTRO=unset
ARG LINUX_DISTRO_VER=unset
```

If this sounds reasonable, happy to send a 2nd PR with this change

---

Yes in my opinion, this is a pattern that should be used in any Dockerfile where it's unacceptable to silently fall back to default values if configuration is not passed.

I just didn't want to go any further with it in this PR and grow the diff here.

I'd support a 2nd PR with this change.